frontend/src/components/licences/LicenceDetails.vue

```diff
@@ -23,7 +23,7 @@
             <AppLabel>Health Authority:</AppLabel>
           </v-col>
           <v-col cols="12" sm="6" class="mt-1">
-
+            {{ getHealthAuthorityNameById(licence?.healthAuthorityId) ? getHealthAuthorityNameById(licence?.healthAuthorityId) : BLANK_FIELD }}
```

Comment:
Because it's calling getHealthAuthorityNameById twice I'd probably make this a computed method.

Reply:
it is a great suggestion. code is updated. thanks!!

Reply:
Much cleaner :)